classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
```

Comment:
- [ ] This method should add `student` to the student array to implement the `has-many` of the relationship between `classroom` and `student`, something like so: 

  def add_student(student)
    @students << student
    student.classroom = self
  end

Reply:
@Mihndim2020 Thanks I will do that.